src/services/parser/__test__/ehr-request-parser.test.js

```diff
@@ -0,0 +1,28 @@
+import path from "path";
+import expect from "expect";
+import { readFileSync } from "fs";
+import { parseEhrRequestMessage } from "../ehr-request-parser";
+import { validateFieldsHaveSuccessfullyParsed } from "../parsing-validation";
+
+// Mocking
+jest.mock("../parsing-validation");
+
+describe('ehr-request-parser.js', () => {
+  it('should parse successfully given an ehr request', async () => {
```

Comment:
Maybe should rename to `should parse an ehr request successfully` so that it's consistent with the naming of other parsing tests.